### ISSUE TITLE
improvements to gossfile configuration format

### ISSUE THREAD

**Issue by sshipway** [aelsabbahy/goss#696]
**Describe the feature:**
Currently, the way to define included gossfiles feels inflexible and inconsistent.  I would like to improve this while retaining backwards compatibility with the existing configuration format.

**Describe the solution you'd like**
Currently, we use this format to inclue a gossfile:
```
gossfile:
  goss_httpd.yaml: {}
```

I would like this to have optional attributes, so that it can be expressed as:
```
gossfile:
  xxxxx:
    file: goss_httpd.yaml
    skip: false
```

The `file` attribute would override the keyval as the filename/wildcard to be included.  The optional `skip` attribute would allow this include to be disabled (as with other `skip` attributes on other resource types)

**Describe alternatives you've considered**
The current method works, but is awkward if you want to use skip to disable a whole gossfile rather than a single resource.  Also, having the filename in the keyval rather than as an attribute can make for some large and unweildy keyvals

**Comment by sshipway** [comment]
I have a created pull request for this in #697

**Comment by sshipway** [comment]
-bump-

**Comment by aelsabbahy** [comment]
Closed by #697
